Dockerfile

```diff
@@ -56,15 +66,9 @@ RUN if [ "${FFMPEG}" = "true" ]; then \
     apt-get install -y ffmpeg \
     ; fi
 
-WORKDIR /build
-
-COPY . .
-RUN make prepare-sources
```

Comment:
instead of redoing this work I just did `FROM builder` above to preserve the `make prepare-sources` it did - does that work? what was the original intention in re-doing it here?

Reply:
it's to prepare sources in the image to avoid any download on start. the `REBUILD` allows to bypass rebuilding on start in the `entrypoint.sh` file, as building on start is the default behavior.

I also find it handy as in this way we also keep the source closer to the tagged release.

Reply:
so, because I changed L57 to `FROM builder` it will inherit the results of `make prepare` that was run in the `builder` image, which will still avoid any downloads on start, but will make the `docker build` faster